app/src/main/java/com/dcrandroid/dialog/send/SendDialog.kt

```diff
@@ -338,6 +349,22 @@ class SendDialog(val fragmentActivity: FragmentActivity, dismissListener: Dialog
             }
         }
     }
+
+    override fun onPause() {
+        super.onPause()
+        if(!destinationAddressCard.isSendToAccount){
+            addressG = destinationAddressCard.addressInputHelper.editText.text.toString()
+        }
+        savedInstanceState?.putString("Address", addressG)
+    }
+
+    override fun onSaveInstanceState(savedInstanceState: Bundle) {
+        super.onSaveInstanceState(savedInstanceState)
+        if(!destinationAddressCard.isSendToAccount){
+            addressG = destinationAddressCard.addressInputHelper.editText.text.toString()
```

Comment:
Do not save the address. EditText content(address & amount) is persisted by default since you're reusing the dialog instance. What you should save is the state of "Send to account/address" and the selected account. The video below shows that the selected account gets reset after closing the page.

<img src="https://user-images.githubusercontent.com/19960200/79928359-a5a14280-843a-11ea-90e6-b95d1eb35f55.gif" height="500">